src/main/java/alns/GreedyInsertion.java

```diff
@@ -0,0 +1,53 @@
+package alns;
+
+import data.Problem;
+import objects.Order;
+import subproblem.SubProblem;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+public class GreedyInsertion {
+
+    public static Solution getGreedyInsertion(Solution partialSolution, Order order) {
+        List<Solution> feasibleInsertions = Construction.getAllFeasibleInsertions(partialSolution,order);
+        if (feasibleInsertions.isEmpty()) return null;
+
+        double bestObjective = Double.POSITIVE_INFINITY;
+        partialSolution.setFitness(bestObjective);
```

Comment:
Trengs denne?

Reply:
`if (feasibleInsertions.isEmpty()) return null;` eller alt fra 15-18? 

17-18 legger jeg inn for å kunne oppdatere `bestSolution` og `bestObjective` senere, men mulig det ikke trengs?

Reply:
Kommentaren er til linje 18. Dersom det er flere linjer tror jeg det står i kommentaren. Poenget er bare at fitness instansieres til Double.POSITIVE_INFINITY når objektet opprettes.

Reply:
Yes, det gir mening! Fjerner'n ;)